src/test/unit/StrategyManagerUnit.t.sol

```diff
@@ -463,6 +525,10 @@ contract StrategyManagerUnitTests is Test, Utils {
         cheats.startPrank(strategyManager.owner());
         IStrategy[] memory _strategy = new IStrategy[](1);
         _strategy[0] = IStrategy(address(reenterer));
+        for (uint256 i = 0; i < _strategy.length; ++i) {
```

Comment:
these 5 lines are used a lot, feel like we could make them another function

Reply:
we use this 4 times. I think it is OK to not make a function for now.

Reply:
sounds good